tcpdump/scripts/tcpdump.sh

```diff
@@ -5,7 +5,16 @@ export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/sd/lib:/sd/usr/lib
 export PATH=$PATH:/sd/usr/bin:/sd/usr/sbin
 
 MYTIME=`date +%s`
-MYCMD=`cat /tmp/tcpdump.run`
+MYCMD="tcpdump `cat /tmp/tcpdump.run`"
```

Comment:
Have the contents of the 'tcpdump.run' file changed? As far as I'm aware, this file contains the full tcpdump command (including the leading tcpdump).

Reply:
hey @sebkinne - Great Question. If you look at my initial comments it points to a forum post where tcpdump is broken. I had same problems on new nano which is why I did this change for the original error on fresh install of " /pineapple/modules/tcpdump/scripts/tcpdump.sh: eval: line 1: -w: not found " -- i am not familiar with original code if 'cat /tmp/tcpdump.run' was supposed to be causing tcpdump to run, i can say it does not and is quite broken as is.

Reply:
https://github.com/hak5/wifipineapple-modules/pull/41